src/app/issuer/components/issuer-create/issuer-create.component.ts

```diff
@@ -14,6 +14,10 @@ import { UserProfileEmail } from '../../../common/model/user-profile.model';
 import { FormFieldSelectOption } from '../../../common/components/formfield-select';
 import { AppConfigService } from '../../../common/app-config.service';
 import { typedFormGroup } from '../../../common/util/typed-forms';
+import { CaptchaService } from '../../../common/services/captcha.service';
+import { TranslateService } from '@ngx-translate/core';
+import 'altcha';
```

Comment:
Somehow this seems to be another import style than usual

Reply:
I think other import styles do not work for this library